src/k8s-engine/lima.ts

```diff
@@ -2001,10 +2001,16 @@ CREDFWD_URL="http://${ hostIPAddr }:${ stateInfo.port }"
 `;
       const credfwdDir = '/etc/rancher/desktop';
       const credfwdFile = `${ credfwdDir }/credfwd`;
+      const configContents = `{
+  "credsStore": "rancher-desktop"
+}
+`;
 
       await this.ssh('sudo', 'mkdir', '-p', credfwdDir);
       await this.writeFile(credfwdFile, fileContents, 0o644);
       await this.writeFile('/usr/local/bin/docker-credential-rancher-desktop', DOCKER_CREDENTIAL_SCRIPT, 0o755);
+      await this.ssh('sudo', 'mkdir', '/root/.docker');
+      await this.writeFile('/root/.docker/config.json', configContents, 0o644);
```

Comment:
What if the file already exists? It may contain other configuration settings that the user wants to persist. So we need to read the file, edit just the `credsStore` setting, and then write it back.

Reply:
I'm not seeing this (at least on lima) -- every time I run rancher-desktop, it recreates the `/root` partition.

Reply:
Because it was not persisted, which is a bug. I've just filed https://github.com/rancher-sandbox/rancher-desktop/pull/2250/files

Reply:
Noted that this isn't happening, waiting for PR #2250